model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -53,6 +53,25 @@ def extract_holder_weights(constant_name, node_target, model, weights, to_numpy)
     return weights
 
 
+def get_node_args_spec(node: Node) -> List[str]:
+    """
+    Return a list of the node's function args & kwargs, so we can insert the kwarg as arg
```

Comment:
give some more explanation, maybe give an example of a problematic case.
the terms "function args", and "insert the kwargs as args" are not clear, I know what you mean just because you explained it to me personally.
So elaborate a little more, something like:
"This function builds an ordered (?) list of layer operation arguments. It flattens and combines both the args list and the kwargs dictionary for a single list, specifically, enables to treat the kwargs as arguments. This is important for the case where functional input (or find a better term), which is a dynamic output of another layer's operation (again, fix my explanation if it is not correct), is passed to the layer as a key argument".

Also, "return**s**", and a '.' at the end of a sentence.